src/main/java/com/vmtapp/enterprise/service/TicketServiceStub.java

```diff
@@ -67,7 +67,15 @@ public TicketServiceStub(ITicketDao ticketDao){
         return ticketsToReturn;
         
     }
+    public List<Ticket> fetchAllTickets(){
+        return ticketDao.fetchAll();
+    }
+    public ArrayList fetchTicketsById(String id){
+        ArrayList<Ticket> ticketsToReturn = new ArrayList<Ticket>();
 
+        return ticketsToReturn;
+
+    }
```

Comment:
For this part, I added are the missing part (some functions and statements)that make the 3 dummy functions work.